platform-sdk/swirlds-platform-core/src/testFixtures/java/module-info.java

```diff
@@ -0,0 +1,15 @@
+open module com.swirlds.platform.test.fixtures {
+    requires com.swirlds.common;
+    requires com.swirlds.common.test.fixtures;
+    requires com.swirlds.platform;
+    requires org.apache.commons.lang3;
+    requires org.junit.jupiter.api;
+    requires com.swirlds.test.framework;
+    requires static com.github.spotbugs.annotations;
```

Comment:
I am not sure if we are allowed include 3rd party libraries with `[LGPL-2.1 license](https://github.com/spotbugs/spotbugs/blob/master/LICENSE)` license.